tests/functional/test_request_pages.py

```diff
@@ -1302,3 +1302,75 @@ def test_request_action_required_alert(
     page.locator(".tree__file").filter(has_text="file1").click()
     expect(content_alert).to_be_visible()
     expect(content_alert_link).to_be_visible()
+
+
+def test_conflicted_file_decision(live_server, context, page, bll):
```

Comment:
This test is fine, but it does duplicate all of the test setup from the unit test, and the only thing it's testing that is different is the html content in the decision tooltip. We could just add an extra bit to the `test_file_vote_buttons` test in the same file, which has a test already for the conflicted case [here](https://github.com/opensafely-core/airlock/blob/main/tests/functional/test_request_pages.py#L528)

So, at the end of that `if` block, logout as the output checker, login as the author, do `decision_locator.hover()` again, and check that it shows different text to the author.